luaui/Widgets_BAR/cmd_select_commander.lua

```diff
@@ -0,0 +1,28 @@
+function widget:GetInfo()
+  return {
+    name = "SelectCommander",
+    desc = "Press ctrl+c to select the commander",
+    author = "Teifion",
+    date = "26/10/2020",
+    license = "GNU GPL, v2 or later",
+    layer = 0,
+    enabled = true
+  }
+end
+
+function widget:Update()
+  local t = Spring.GetGameSeconds()
+  if (select(3,Spring.GetPlayerInfo(Spring.GetMyPlayerID(),false)) or t > 10) then
+    widgetHandler:RemoveWidget(self)
+    return
+  end
+  if (t > 0) then
+    local unitArray = Spring.GetTeamUnits(Spring.GetMyTeamID())
+    for key, unit in pairs(unitArray) do
+      if (unitDef.customParams.iscommander == true) do
```

Comment:
Where does the `unitDef` variable come from?

Reply:
That's a copy paste typo, should have been `unit`. Thanks for the spot.